app/views/camps/_form.erb

```diff
@@ -0,0 +1,301 @@
+<script>
```

Comment:
Hm it looks like we're moving this file to erb (should be `_form.html.erb` I think), but are adding other .haml files as well in this PR. Is this intention to remove HAML? Or is there some other reason this needs to be erb now?

Reply:
Yes, good catch. I spent the extra time now to remove all evil haml from the project. Long overdue. Erb is more than enough.